tldraw/apps/tldraw-logseq/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -96,6 +96,13 @@ export const ContextMenu = observer(function ContextMenu({
                 </Button>
               </div>
               <ReactContextMenu.Separator className="menu-separator" />
+              <ReactContextMenu.Item
+                className="tl-menu-item"
+                onClick={() => runAndTransition(app.packIntoRectangle)}
+              >
+                Pack shapes into rectangle
```

Comment:
```suggestion
                Pack into rectangle
```
I think we can remove `shapes` for consistency. All context menu actions are applied to shapes.